### ISSUE TITLE
request: pass ALT text to renderPlot

### ISSUE THREAD

**Issue by howardcoleman** [rstudio/shiny#612]
It would be helpful if renderPlot() were to accept an optional argument (a string) that would become the value of an alt attribute in the resulting image tag:

e.g:

`renderPlot(...,alt="alternative text",...)`

produces

`<img ... alt="alternative text" ...>`

in the output HTML.

Incorporating the alt argument into a list of attributes, as in renderImage(), would also be fine, if it's feasible.

**Comment by olyerickson** [comment]
There is a big need for this feature, esp. for people building Shiny apps that must meet ADA guidelines. Can the priority be raised???

**Comment by bklingen** [comment]
+1

**Comment by pfeiffss** [comment]
I need this, too -- why can't Shiny just send HTML code to the browser??  It's not accessible at all right now.

**Comment by jooyoungseo** [comment]
@pfeiffss, we will close this issue soon via #3006. Thanks for your patience.
